libs/SalesforceSDKCore/SalesforceSDKCore.xcodeproj/project.pbxproj

```diff
@@ -1069,6 +1073,8 @@
 		4F96FC511BFD32130022F021 /* Common */ = {
 			isa = PBXGroup;
 			children = (
+				B7BAFF491D70ADD50020F756 /* NSUserDefaults+SFAdditions.h */,
```

Comment:
Can we move these to be in alphabetical order with the rest of the code files in this folder?